optaplanner-core/src/main/java/org/optaplanner/core/impl/score/stream/bavet/uni/BavetAbstractUniConstraintStream.java

```diff
@@ -140,8 +139,9 @@ public void addChildStream(BavetAbstractUniConstraintStream<Solution_, A> childS
     }
 
     @Override
-    public <GroupKeyA_, GroupKeyB_, ResultContainer_, Result_> TriConstraintStream<GroupKeyA_, GroupKeyB_, Result_> groupBy(
-            Function<A, GroupKeyA_> groupKeyAMapping, Function<A, GroupKeyB_> groupKeyBMapping,
+    public <GroupKeyA_, GroupKeyB_, ResultContainer_, Result_>
+    TriConstraintStream<GroupKeyA_, GroupKeyB_, Result_> groupBy(Function<A, GroupKeyA_> groupKeyAMapping,
```

Comment:
Maybe we should figure out a shorter name for GroupKeyA_ (that would make me and Radovan happy, I am sure Lukas loves any shorter name) so we don't go over the line char limit before hitting the first (